site/content/en/concepts/packaging/_index.md

```diff
@@ -25,4 +25,54 @@ For compatibility with other arbitrary formats, kpt supports generating
 resource configuration data from templates, configuration DSLs, and programs
 using [source functions].
 
+## Composite Packages
+
+A `composite package` is a `kpt` package which contains 1 or more `kpt` packages
+in its directory tree.
+
+Here is an example directory structure of a `composite package`
+
+```sh
+foo # composite package
```

Comment:
Instead of composite package here, I think we can mark this one as `top-level package` and the others as `nested package`?